### PR TITLE
Ported the ACE2 sandbags

### PR DESCRIPTION
**Pull request by ulteq**
This adds PhysX enabled, stackable sandbags.

http://abload.de/img/arma3op_2015_06_02_17tcs7m.jpg
http://abload.de/img/arma3op_2015_06_02_18i3k8f.jpg

### CODE REVIEW

**Comment by PabstMirror** [comment]
Very cool module.

* `deployCancle` spelling

* sometimes the bags forget to physics:
untitled

* The rightMouseButton cancel (`zoomTemp`) never works for me, for explosives we added this as a backup:
```
//Cancel placement if interact menu opened
["interactMenuOpened", {
    if (GVAR(pfeh_running) && {!isNull (GVAR(Setup))}) then {
        call FUNC(place_Cancel)
    };
}] call EFUNC(common,addEventHandler);
```

File: addons/sandbag/functions/fnc_deployCancle.sqf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+if (isNull GVAR(placer)) exitWith {};
+
+[GVAR(deployPFH)] call cba_fnc_removePerFrameHandler;
+
+if (!isNull (GVAR(sandBag))) then {
+    deleteVehicle GVAR(sandBag);
+    GVAR(sandBag) = objNull;
+};
+
+[GVAR(placer), "ACE_Explosives", false] call EFUNC(Common,setForceWalkStatus);
```
Comment (PabstMirror) [inline addons/sandbag/functions/fnc_deployCancle.sqf]: Key should be "ACE_Sandbag"

File: addons/sandbag/functions/fnc_deployCancle.sqf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+GVAR(sandBag) = objNull;
+GVAR(placer) = objNull;
+
+call EFUNC(interaction,hideMouseHint);
+[GVAR(placer), "DefaultAction", GVAR(placer) getVariable [QGVAR(Deploy),  -1]] call EFUNC(Common,removeActionEventHandler);
```
Comment (PabstMirror) [inline addons/sandbag/functions/fnc_deployCancle.sqf]: Probably want to move `GVAR(placer) = objNull;` bellow these lines.

**Comment by ulteq** [comment]
> sometimes the bags forget to physics:

This happend after I moved `GVAR(sandBag) enableSimulationGlobal true;` into the waitAndExecute block.

**Pull request merged**